js/controllers/window.js

```diff
@@ -131,16 +141,9 @@ WindowController.prototype.toggleSidebar_ = function() {
     $('#toggle-sidebar')
         .attr('title', chrome.i18n.getMessage('openSidebarButton'));
   } else {
-    this.settings_.set('sidebaropen', true);
-    $('#sidebar').css('width', this.settings_.get('sidebarwidth') + 'px');
-    $('#sidebar').css('border-right-width', '2px');
-    $('#toggle-sidebar')
-        .attr('title', chrome.i18n.getMessage('closeSidebarButton'));
+    this.openSidebar();
   }
   this.editor_.focus();
-  setTimeout(function() {
```

Comment:
How come we don't need this anymore?

Reply:
Git blame says the `resize` event was added a long time ago: https://github.com/GoogleChromeLabs/text-app/commit/3a050ecbdf2a191fdcfea2ce899a40ba68450a35#diff-414ecf1c421030737f8f5542793289376daad4a4341ac5c8e50c0cbad6564234

The other part of that commit that uses the event doesn't exist anymore, and there are no search results for `'resize'`.